src/adapters/adapter.js

```diff
@@ -0,0 +1,6 @@
+function adapterByOrigin() {
+  if (location.origin == DOMAINS.GITHUB)   
+    return new GitHub()
```

Comment:
Expected '{' and instead saw 'return'.<br>Missing semicolon.<br>'GitHub' is not defined.